### PR TITLE
Feature/no mongo

### PR DESCRIPTION
**Pull request by whatgwan**

### CODE REVIEW

File: installer [hunk 1]
```diff
@@ -1140,7 +1160,8 @@ sub run_hooks
 				$ENV{UNATTENDED} = 1 if ($noninteractive);
 				$ENV{SIMULATE} = 1 if ($simulate);
 				$ENV{LOGFILE} = $logfile if (!$simulate);
-				$ENV{CLEANSLATE} = 1 if ($is_initial_install);
+				$ENV{CLEANSLATE} = ($is_initial_install);
```
Comment (markdueck1) [inline installer]: this needs to be removed

File: installer_hooks/05-postcopy-configfiles [hunk 2]
```diff
@@ -5,7 +5,7 @@
 SCRIPTPATH=${0%/*}
 . $SCRIPTPATH/common_functions.sh
 
-if [ -n "$CLEANSLATE" ]; then
+if [ -z "$CLEANSLATE" ]; then
```
Comment (markdueck1) [inline installer_hooks/05-postcopy-configfiles]: this needs to be removed

File: installer_hooks/10-postcopy-confmerges [hunk 3]
```diff
@@ -5,7 +5,7 @@
 SCRIPTPATH=${0%/*}
 . $SCRIPTPATH/common_functions.sh
 
-if [ -n "$CLEANSLATE" ]; then
+if [ -z "$CLEANSLATE" ]; then
```
Comment (markdueck1) [inline installer_hooks/10-postcopy-confmerges]: this needs to be removed

File: installer_hooks/60-postcopy-upgrades [hunk 4]
```diff
@@ -6,7 +6,7 @@
 SCRIPTPATH=${0%/*}
 . $SCRIPTPATH/common_functions.sh
 
-if [ -n "$CLEANSLATE" ]; then
+if [ -z "$CLEANSLATE" ]; then
```
Comment (markdueck1) [inline installer_hooks/60-postcopy-upgrades]: this needs to be removed

File: installer_hooks/99-postinstall-welcome [hunk 5]
```diff
@@ -6,7 +6,7 @@ SCRIPTPATH=${0%/*}
 . $SCRIPTPATH/common_functions.sh
 
 # an initial install of this product
-if [ -n "$CLEANSLATE" ]; then
+if [ -z "$CLEANSLATE" ]; then
```
Comment (markdueck1) [inline installer_hooks/99-postinstall-welcome]: this needs to be removed

**Review by markdueck1 — CHANGES_REQUESTED** [review]

File: installer_hooks/99-postinstall-welcome (hunk 5 above)
Comment (whatgwan) [reply installer_hooks/99-postinstall-welcome]: I have reverted the change

File: installer_hooks/60-postcopy-upgrades (hunk 4 above)
Comment (whatgwan) [reply installer_hooks/60-postcopy-upgrades]: reverted

File: installer_hooks/10-postcopy-confmerges (hunk 3 above)
Comment (whatgwan) [reply installer_hooks/10-postcopy-confmerges]: reverted

File: installer_hooks/05-postcopy-configfiles (hunk 2 above)
Comment (whatgwan) [reply installer_hooks/05-postcopy-configfiles]: reverted

File: installer (hunk 1 above)
Comment (whatgwan) [reply installer]: reverted

**Pull request closed without merging**